src/seedu/addressbook/ui/TextUi.java

```diff
@@ -46,6 +47,18 @@ public TextUi(InputStream in, PrintStream out) {
         this.in = new Scanner(in);
         this.out = out;
     }
+    
+    /**
+     * Returns a concatenated version of the printable strings of each object
+     */
+    
+    public String getPrintableString(Printable... printables) {
+        final StringBuilder formatted = new StringBuilder();
+        for (Printable x : printables) {
+            formatted.append(x.getPrintableString()).append('\n');
```

Comment:
Doesn't this result in an extra `,` at the end? You could have used `StringJoiner` or `String.join` instead.